SPECS/python-h5py/python-h5py.spec

```diff
@@ -92,6 +92,11 @@ cd -
 %{python3_sitearch}/%{name}-%{version}-*.egg-info
 
 %changelog
+* Mon May 20 2024 George Mileka <gmileka@microsoft.com> - 3.7.0-5
+- Bumping release version so that the newer 1.14.4 hdf5 package is picked
+  during the build. The new 1.14.4 hdf5 generates shared objects with .310
+  and .311 suffixes (replacing the older .200 suffixed ones).
```

Comment:
Nit-pick on comment clarity: it's not about the name of the file containing the library but the versions of the libraries the binaries link to. The library names along with their versions are embedded into the binary and these version numbers happen to match the number in the file name but that's just convention.

Reply:
Makes sense. I didn't not know the versions are embedded in the 'import' libraries. I've updated the comment.